templates/csharp/sso-tab/Components/Graph.razor

```diff
@@ -77,12 +80,13 @@
         IsLoading = true;
         try
         {
-            await teamsUserCredential.GetTokenAsync(new TokenRequestContext(new string[] { _scope }), new System.Threading.CancellationToken());
+            var graph = await GetGraphServiceClient();
+            Profile = await graph.Me.GetAsync();
```

Comment:
can we just use OBOCredential to acquire a graph token instead of calling graph API here? This code works but may not easy to understand if user does not have context about what we're tring to do (get a graph token) here.

Reply:
Updated. Now we are using masl.net to get graph token with specified scope.